DictionaryTypes.txt

```diff
@@ -274,6 +274,8 @@ enum(ch_type) "Type of the channel"
     dipole_wave    1000		     "Dipole time curve"
     goodness_fit   1001		     "Goodness of fit"
     fnirs	   1100		     "Functional near-infrared spectroscopy"
+    temperature	   1200		     "Temperature"
+    galvanic	   1300		     "Galvanic skin response"
```

Comment:
We should name the channel type eda and the description should be Electrodermal activity. GSR is an outdated term that ideally should also be eliminated from BIDS....

See also Wikipedia in the naming:

https://en.wikipedia.org/wiki/Electrodermal_activity?wprov=sfti1

Reply:
I feel quite strongly we should use EDA. I haven't read GSR in publications in quite a while 

(But of course my literature selection is biased)

Reply:
@cbrnr @sappelhoff any thoughts?

Reply:
I am not so deeply entrenched in this nomenclature, but if the info in the wikipedia article is valid that "EDA" is a term to standardize the diverse previous terms, then I am all for it.

On the BIDS side we can (given the assumption is true that EDA is the new standard) then deprecate GSR and declare that people use EDA for future datasets.

In MNE-BIDS we can map both EDA and GSR (bids) to EDA (mne), and warn if we do a GSR->EDA mapping, that GSR is deprecated in BIDS.

Reply:
I'll ask some colleagues of mine who do (or did) peripheral measurements of anything "emotion", maybe they have an opinion

Reply:
tl;dr: Both terms are used interchangeably, but there seems to be a consensus that EDA is to be preferred these days

So it seems that EDA and GSR are used interchangeably, but GSR mostly sticks around for historical reasons. In many publications, you'll find phrases like "... measured electrodermal activity (EDA; also known as galvanic skin response, GSR)". Like, this kind of double-naming appears to be kind of omnipresent.

I checked how software packages for EDA processing are named:
- [LEDALAB](https://www.mathworks.com/matlabcentral/fileexchange/68502-ledalab-open-source-matlab-software-for-analysis-of-skin-conductance-data-viz-eda-gsr)
- [pyEDA](https://github.com/HealthSciTech/pyEDA)
- NeuroKit2 has an [`eda` module](https://neuropsychology.github.io/NeuroKit/functions/eda.html)

In summary, my favorite would be `eda`, but I could live with `gsr` too – the community will understand both.

I veto against `galvanic`, though, as this is too generic, and I suppose it won't ring a bell for many folks.

Reply:
+1 for `gsr` then, as it'll make our lives a bit easier (we wouldn't have to discuss deprecating `gsr` in favor of `eda` at BIDS) and it seems to stick.

Reply:
Let's do `gsr` then, this also avoids @agramfort's brain freeze 😅